### PR TITLE
Next dev2018 09 into Master (David, please merge - thank you)

### PR DESCRIPTION
**Pull request by bote-reginald**
- ShipYardDockView adapted to ShipsUnderConstruction (get the image folder from Dropbox!)
- optimized a little bit these files FromTechObj-xx_(autoCreated).csv

### CODE REVIEW

**Comment by bote-reginald** [comment]
would be nice to get merged latest stuff as well (5 minutes ago)

File: SupremacyCore/Combat/CombatEngine.cs [hunk 1]
```diff
@@ -408,30 +408,30 @@ protected bool WasRetreateSuccessful(CombatUnit unit, bool oppositionIsRushing,
             if (weaponRatio > 6) // if you rush and outgun the retreater they are less likely to get away
             {
                 retreatChanceModifier = -30;
-                GameLog.Print("Weapon ratio was {0). -30 modifier", weaponRatio);
+                GameLog.Print("Weapon ratio was {0}. -30 modifier", weaponRatio);
```
[the remark this answers is not in the document]
Comment (KennethGF) [reply SupremacyCore/Combat/CombatEngine.cs]: done

File: SupremacyUI/SystemProductionPanel.cs [hunk 2]
```diff
@@ -629,14 +636,17 @@ private void slider_ActiveUnitsChanged(object sender, DependencyPropertyChangedE
                         var activateCommand = Model.ActivateFacilityCommand;
                         if ((activateCommand != null) && activateCommand.CanExecute(category))
                             activateCommand.Execute(category);
+                        GameLog.Print("slider_ActiveUnitsChanged... category {1} IN-CREASED {0}", delta, category);
```
[the remark this answers is not in the document]
Comment (KennethGF) [reply SupremacyUI/SystemProductionPanel.cs]: done

File: SupremacyUI/SystemProductionPanel.cs [hunk 3]
```diff
@@ -629,14 +636,17 @@ private void slider_ActiveUnitsChanged(object sender, DependencyPropertyChangedE
...
                     }
                     else
                     {
                         var deactivateCommand = Model.DeactivateFacilityCommand;
                         if ((deactivateCommand != null) && deactivateCommand.CanExecute(category))
                             deactivateCommand.Execute(category);
+                        GameLog.Print("slider_ActiveUnitsChanged... category {1} DE-CREASED {0}", delta, category);
```
[the remark this answers is not in the document]
Comment (KennethGF) [reply SupremacyUI/SystemProductionPanel.cs]: done

File: SupremacyClientComponents/Themes/Borg/ShipyardDockView.xaml [hunk 4]
```diff
@@ -66,14 +66,15 @@
...
+    <Setter Property="Width" Value="144" />
+    <Setter Property="Height" Value="120" />
     <Setter Property="Template">
       <Setter.Value>
         <ControlTemplate TargetType="{x:Type Button}">
- <!--image of ship under construction-->
-          <Border x:Name="ShipBorder" BorderBrush="White" BorderThickness="2" CornerRadius="3" Background="#AF000000">
-            <Image x:Name="ShipImage" Width="84" Stretch="Uniform" HorizontalAlignment="Center" VerticalAlignment="Center"
+ <!--image of ship under construction--><!--
+          <Border x:Name="ShipBorder" BorderBrush="White" BorderThickness="2" CornerRadius="3" Background="#AF000000">-->
```
[the remark this answers is not in the document]
Comment (KennethGF) [reply SupremacyClientComponents/Themes/Borg/ShipyardDockView.xaml]: done

File: SupremacyUI/SystemProductionPanel.cs [hunk 5]
```diff
@@ -740,6 +750,16 @@ private void ResetSliders()
 
                 _sliderGroup.ResetPool(colony.Population.CurrentValue);
 
+                //bool _traceSliders = true;
```
[the remark this answers is not in the document]
Comment (KennethGF) [reply SupremacyUI/SystemProductionPanel.cs]: Done

**Comment by bote-reginald** [comment]
Thank you for improving my badly done code :-)

**Pull request merged**